mcs/class/Mono.Security/Mono.Security.X509/PKCS12.cs

```diff
@@ -383,8 +383,13 @@ private void Decode (byte[] data)
 
 				byte[] authSafeData = authSafe.Content [0].Value;
 				byte[] calculatedMac = MAC (_password, macSalt.Value, _iterations, authSafeData);
-				if (!Compare (macValue, calculatedMac))
-					throw new CryptographicException ("Invalid MAC - file may have been tampered!");
+				if (!Compare (macValue, calculatedMac)) {
```

Comment:
I don't think they should be in here.

@nealef to avoid this in the future you should push your changes to a new branch and open the PR from there, instead of working on `master` (GitHub will update a PR when new commits are pushed to a branch).

@tritao can you please cherry-pick the commit?